pages/operator-address-select.js

```diff
@@ -22,25 +22,33 @@ const OperatorAddressLookup = (props) => (
           <Heading as="h2" size="MEDIUM">
             {props.t("Postcode")}
           </Heading>
-          <Paragraph className="operatorPostcodeDisplay" mb={0}>
-            {props.cumulativeFullAnswers.operator_postcode_find}
-          </Paragraph>
-          <Link
-            id="changeOperatorPostcode"
-            href={
-              props.editModeFirstPage
-                ? "/operator-address?edit=operator-address"
-                : "/operator-address"
-            }
-          >
-            {props.t("Change postcode")}
-          </Link>
+          <d1>
+            <dt>
+              <Paragraph className="operatorPostcodeDisplay" mb={0}>
```

Comment:
I'd have them either both as dt or both as dd, looks weird in this context not having them aligned.

Reply:
hmm, don't think that's how description list work. They need both a dt (that defines terms/names) and a dd tag(describes each term/name). Wouldn't make sense to do 2 of both within a description list

Reply:
Yeah fair enough, seems for screen readers it makes a lot of sense to have the change postcode as a dd of the postcode dt too.